### PR TITLE
Doc update on the azure-API which is present in node integration Tab

### PR DESCRIPTION
**Pull request by shaik80**
### :nut_and_bolt: Description: What code changed, and why?
This PR https://github.com/chef/automate/pull/5257 allow the user to add subscription Id in the azure-API section, This changed document say user about subscription ID
### :chains: Related Resources
https://github.com/chef/automate/issues/5252
### :+1: Definition of Done
- Made changes in the node-manager service Readme file for dev understanding 
- Done changes in docs-chef-io service for customer understanding 

### :athletic_shoe: How to Build and Test the Change
- goto **components/docs-chef-io/**
- run command **make serve**

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable

### CODE REVIEW

File: components/nodemanager-service/README.md [hunk 1]
```diff
@@ -101,7 +101,9 @@ To create a node manager instance, the user can call NodeManagerService.Create w
 ```
 For aws managers, the credential added must have key/values of `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.  Users running the binary in aws can use the environmental creds to connect to the sdk instead of the access creds, by simply leaving out credential data when creating the manager.
 
-For azure managers, the credential added must have key/values of `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, and `AZURE_TENANT_ID`.
+For azure managers, the credential added must have key/values of `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE_TENANT_ID`,`AZURE_SUBSCRIPTION_ID`.
+
+**Note :** If we are using azure-api we can add `AZURE_SUBSCRIPTION_ID`,its optional
```
Comment (vivekshankar1) [inline components/nodemanager-service/README.md:106]: For azure managers, 

VM:
- The credential added must have key/values of `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET` and `AZURE_TENANT_ID`

API:
- The credential added must have key/values of `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET` and `AZURE_TENANT_ID` and `AZURE_SUBSCRIPTION_ID`.
- **Note :** `AZURE_SUBSCRIPTION_ID` is optional

File: components/docs-chef-io/content/automate/node_integrations.md [hunk 2]
```diff
@@ -156,7 +156,7 @@ We do not support Azure Government Cloud.
 When creating an Azure VM Node Manager, you will be required to provide:
 
 1. A name for your manager
-1. Your Azure credentials (client ID, client secret, and tenant ID)
+2. Your Azure credentials (client ID, client secret and tenant ID)
```
Comment (vivek-yadav) [inline components/docs-chef-io/content/automate/node_integrations.md:159]: add Subscription ID

Comment (shaik80) [reply components/docs-chef-io/content/automate/node_integrations.md:159]: we don’t need to add a Subscription ID here, Subscription ID will be support for azure-api not for VM

Comment (vivek-yadav) [reply components/docs-chef-io/content/automate/node_integrations.md:159]: ok

File: components/docs-chef-io/content/automate/node_integrations.md [hunk 3]
```diff
@@ -196,9 +196,13 @@ When creating an Azure API Node Manager, you will be required to provide:
 
 1. A name for your manager
 2. Your Azure credentials (client ID, client secret, and tenant ID)
+3. SubscriptionID is optional
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/node_integrations.md:199]:
```suggestion
3. Subscription ID (optional)
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]
Small change.

File: components/docs-chef-io/content/automate/node_integrations.md (hunk 2 above)
Comment (kagarmoe) [reply components/docs-chef-io/content/automate/node_integrations.md:159]:
```suggestion
2. Your Azure credentials (client ID, client secret, and tenant ID)
```

Unless the user needs to enter the client secret and the tenant ID _together_, then this sentence needs a serial comma.

**Review by kagarmoe — CHANGES_REQUESTED** [review]

File: components/docs-chef-io/content/automate/node_integrations.md (hunk 2 above)
Comment (kagarmoe) [reply components/docs-chef-io/content/automate/node_integrations.md:159]: @shaik80 If you make these changes, then we can merge this PR.

Comment (kagarmoe) [reply components/docs-chef-io/content/automate/node_integrations.md:159]: @vivek-yadav I need to un-resolve this conversation so the changes are visible.

File: components/docs-chef-io/content/automate/node_integrations.md [hunk 4]
```diff
@@ -196,9 +196,13 @@ When creating an Azure API Node Manager, you will be required to provide:
...
 
 This information is required to detect all subscriptions available to your Azure account. Chef Automate creates a nodes reference for each subscription in your account.
 
+![Chef Automate Create API-Azure Manager](/images/automate/Node-Integrations-api-Azure.png)
+
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/node_integrations.md:204]:
```suggestion
```

**Comment by kagarmoe** [comment]
@vivek-yadav @kalroy What should we do with this PR?

**Review by kagarmoe — APPROVED** [review]

**Pull request merged**